packages/mdl-menu/util.js

```diff
@@ -0,0 +1,140 @@
+/**
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+let storedTransformPropertyName_;
+
+/**
+ * Choose the correct transform property to use on the current browser.
+ * @param {Object} globalObj The global object, useful for testing (defaults to window).
+ * @return {string} The transform property name.
+ */
+function getTransformPropertyName(globalObj = window) {
+  if (storedTransformPropertyName_ === undefined || globalObj !== window) {
```

Comment:
I think we should omit this `globalObj !== window` check and the `globalObj === window` check below. I imagine this was done for testing purposes? It may be better, instead, to pass in a `forceRefresh = false` param which you could use within tests.